### ISSUE TITLE
[EROR] Error while posting tweet: File is too big, must be less than 3072kb

### ISSUE THREAD

**Issue by gusuu1** [corbindavenport/tootbot#94]
A lot of posts that I my bot mirrors are too big for Twitter, maybe is there some option to resize the image before uploading it to Twitter?

**Comment by saucylegs** [comment]
Yeah, I also just had this problem, I tweeted Corbin about it and it's because Twitter has a 3MB limit on their API. When I have the time, I think I'll try to integrate an image compressor into Tootbot myself. I suggest Corbin look into this too though

**Comment by gusuu1** [comment]
> Yeah, I also just had this problem, I tweeted Corbin about it and it's because Twitter has a 3MB limit on their API. When I have the time, I think I'll try to integrate an image compressor into Tootbot myself. I suggest Corbin look into this too though

Did you manage to create it?

**Comment by gusuu1** [comment]
Since the project doesn't get updated anymore, I'll figure it out myself

**Comment by gusuu1** [comment]
Turns out this wouldn't be too hard to fix, can this be merged? https://gist.github.com/ShantanuJoshi/23ac55479ab9a613230bd9467d080f33
